doc/source/reference/slycot_outer.rst

```diff
@@ -0,0 +1,87 @@
+.. _function-ref:
+
+************************
+Outer function reference
+************************
+
+.. automodule:: slycot
+   :no-members:
+   :no-inherited-members:
+   :no-special-members:
+
+Analysis
+========
+
+.. autosummary::
+   :toctree: generated/
+
+   ab01nd
```

Comment:
So wee need to add entries here for every additional wrapper? Better write a check that we don't forget anything.

Reply:
Good point. I also thought about that.

There is a python script `/doc/create_names_for_reference.py`  that generates all `slycot._wrapper` and `slycot`
 routines.

I think there are 2 options:
1. a script that compares the output from the script above with the routine names found in `slycot_outer.rst` and `slycot_inner.rst`.
2.  a script that generates the whole files of `slycot_outer.rst` and `slycot_inner.rst`.

I have to look into that.